frontend/data/themes/Yami.obt

```diff
@@ -651,6 +651,10 @@ QDockWidget::float-button {
     border-radius: var(--border_radius);
     background: transparent;
     margin-right: 1px;
+    min-width: var(--icon_base);
+    min-height: var(--icon_base);
+    icon-size: var(--icon_base);
+    padding: var(--padding_base);
```

Comment:
It might make more sense to eliminate padding around the icon (contents) within the float-button widget.